docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -39,6 +39,11 @@ The ``ALTER TABLE EXECUTE`` statement followed by a ``command`` and
 parameters. ``ALTER TABLE EXECUTE`` supports different commands on a
 per-connector basis.
 
+Note that ``=>`` is the symbol used for passing parameters to a specified command.
```

Comment:
should this be called a symbol?

"syntax"? "operator"?

@martint  ?

Reply:
I think "operator", my two cents. Symbol isn't very descriptive.